mbs-s0003-20230501/README.md

```diff
@@ -0,0 +1,140 @@
+* The first axis is the frequency band
+* The second (singleton) axis follows `mnms` convention (reserved for map splits; since each simulation corresponds to one map split, the dimension of this axis is 1)
+* The third axis is Stokes component (I, Q, U)
+* The last axes are the map pixels
+
+The geometry (including shape) of the maps is given in the FITS header:
+```
+SIMPLE  =                    T / conforms to FITS standard                      
+BITPIX  =                  -32 / array data type                                
+NAXIS   =
+ 5 / number of array dimensions                     
+NAXIS1  =                10800                                                  
+NAXIS2  =                 2640
+NAXIS3  =                    3                                                  
+NAXIS4  =                    1                                                  
+NAXIS5  =                    2                                                  
+WCSAXES =                    2 / Number of coordinate axes                      
+CRPIX1  =             5400.625 / Pixel coordinate of reference point            
+CRPIX2  =               1890.5 / Pixel coordinate of reference point            
+CDELT1  =   -0.033333333333333 / [deg] Coordinate increment at reference point  
+CDELT2  =    0.033333333333333 / [deg] Coordinate increment at reference point  
+CUNIT1  = 'deg'                / Units of coordinate increment and value        
+CUNIT2  = 'deg'                / Units of coordinate increment and value        
+CTYPE1  = 'RA---CAR'           / Right ascension, plate caree projection        
+CTYPE2  = 'DEC--CAR'           / Declination, plate caree projection            
+CRVAL1  =                  0.0 / [deg] Coordinate value at reference point      
+CRVAL2  =                  0.0 / [deg] Coordinate value at reference point      
+LONPOLE =                  0.0 / [deg] Native longitude of celestial pole       
+LATPOLE =                 90.0 / [deg] Native latitude of celestial pole        
+MJDREF  =                  0.0 / [d] MJD of fiducial time                       
+RADESYS = 'ICRS'               / Equatorial coordinate system                   END
+```
+
+## Example usage
+
+We give some minimum working examples:
+```python
+import numpy as np
+from pixell import enmap, curvesky
+from os.path import join as opj
+
+# first get some basic info like the path and the filename template
+fdir = '/global/cfs/cdirs/sobs/v4_sims/mbs/mbs-s0003-20230501/sims'
+fbase_template = 'so_scan_s0003_fdw_{bands}_lmax5400_4way_set{split_num}_noise_sim_map{sim_num}.fits'
+
+# let's load the sim for e.g. f150, split 2, sim 123
+bands = 'mf_f090_mf_f150'
+band_idx = 1                # f090 is 0, f150 is 1
+split_num = 2
+sim_num = str(123).zfill(4) # has 4 digits (leading 0's)
```

Comment:
no need to zero-pad here if you zero-pad in the template

Reply:
Nice! Didn't realize this was possible, much cleaner.